tasks/clean_cert.rb

```diff
@@ -56,7 +57,8 @@ def clean_cert(agent)
 
 else
 
-  agents = ENV['PT_agent_certnames'].split(',')
+  params = JSON.parse(STDIN.read)
```

Comment:
This is backwards compatible with the existing comma-separated input, right?

Reply:
Yes, Line 61 ensures that is compatible with the existing comma-separated input.

The only way that it would not be compatible is if someone passes in agent list from an environment variable before running the bolt task.  But that doesn't seem to be possible.

```
% PT_agent_certnames="agent1,agent2,agent3"; echo $PT_agent_certnames; bolt task run purge_node -t localhost
agent1,agent2,agent3
Could not find a task named "purge_nodes". For a list of available tasks, run "bolt task show"
```